lib/tapioca/dsl/helpers/active_record_column_type_helper.rb

```diff
@@ -98,11 +98,13 @@ def column_type_for(column_name)
         sig { params(column_type: T.untyped).returns(String) }
         def type_for_activerecord_value(column_type)
           case column_type
-          when defined?(MoneyColumn) && MoneyColumn::ActiveRecordType
+          when ->(type) { defined?(MoneyColumn) && MoneyColumn::ActiveRecordType === type }
```

Comment:
Interesting pattern, which answers the question:

> Why does `Proc` respond to `#===`?! When would you ever call `some_proc === some_arg` instead of just `some_proc.call(some_arg)`?!

Reply:
Yes, a lot of the `#===` implementations are so that you can use them in `when` clauses.